cdp_scrapers/instances/portland-static.json

```diff
@@ -0,0 +1,94 @@
+{
+    "persons": {
+        "Ted Wheeler": {
+            "name": "Ted Wheeler",
+            "is_active": true,
+            "router_string": null,
+            "email": "wheelermediarequests@portlandoregon.gov",
+            "phone": "503-823-4120",
+            "website": "https://www.portland.gov/wheeler",
+            "picture_uri": "https://www.portland.gov/sites/default/files/styles/4x5_320w/public/2020-03/mayor-wheeler-headshot-4x5-3-5-2020.jpg",
+            "seat": {
+                "name": "Mayor",
+                "electoral_area": "Portland",
+                "electoral_type": null,
+                "image_uri": "https://www.portland.gov/sites/default/files/styles/facebook/public/2020-01/city_hall_portland_oregon_2012.jpg",
+                "external_source_id": null,
+                "roles": null
+            },
+            "external_source_id": null
+        },
+        "Carmen Rubio": {
+            "name": "Carmen Rubio",
+            "is_active": true,
+            "router_string": null,
+            "email": "comm.rubio@portlandoregon.gov",
+            "phone": "503-823-3008",
+            "website": "https://www.portland.gov/rubio",
+            "picture_uri": "https://www.portland.gov/sites/default/files/styles/4x5_320w/public/2021/carmen-rubio.jpg",
+            "seat": {
+                "name": "Position 1",
+                "electoral_area": "Portland",
```

Comment:
Interesting. It's like the positions are not tied to any districts. I can't find any district maps or anything like that. So, I get why `electoral_area` is the same for everyone. But this file is already for the Portland scraper, so saying Portland here, to me, is redundant. What do you think about `Citywide`? That's what we use for e.g. Position 9 in Seattle. I'll leave it up to you.

Reply:
I changed all the electoral areas to `Citywide`. You can find a list of positions in the Portland City Commission here: https://ballotpedia.org/Portland,_Oregon.

I took the images for `image_url` from the webpages of each Commissioner. For example, I found Commissioner Carmen Rubio's `image_uri` on her webpage: https://www.portland.gov/rubio.

Reply:
Shak I was referrting to `seat.image_uri`, not `Person.picture_uri`. I understand where you got `Person.picture_uri`. Anyway, you've answered my question; looks like you were also unable to find out what region corresponds to what position number. So every person's `seat.electroral_area` is now `Citywide`. Let's go with that for now.

Reply:
If you go to a mayor/commissioner's webpage and scroll down, you will find a set of images that link to the offices the mayor/commissioner oversees. I used these pictures for each `seat.image_uri`, and I can replace them if you have a better idea.

Reply:
Ah got it! That's awesome Shak. Great find.